modules/federated_language/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "homepage": "https://github.com/google-parfait/federated-language/",
+  "maintainers": [
+    {
+      "email": "federated-language-build@google.com",
+      "name": "Federated Language Team"
+    },
+    {
+      "name": "Michael Reneer",
+      "email": "michaelreneer@google.com",
+      "github": "michaelreneer"
+    }
+  ],
+  "repository": ["github:google-parfait/federated-language"],
+  "versions": [],
```

Comment:
0.1.0 should be listed here.

Reply:
I can add that, strange that the tool did not add this. Any idea why?